next.config.js

```diff
@@ -0,0 +1,63 @@
+const createMDX = require('@next/mdx');
+const remarkPrism = require('remark-prism');
+
+/** @type {import('next').NextConfig} */
+const nextConfig = {
+  pageExtensions: ['js', 'jsx', 'mdx', 'ts', 'tsx'],
+
+  trailingSlash: true,
+
+  experimental: {
+    serverComponentsExternalPackages: ['rehype-preset-minify'],
+  },
+};
+
+const withMDX = createMDX({
+  options: {
+    remarkPlugins: [remarkPrism],
+  },
+});
+
+module.exports = withMDX(nextConfig);
+
+// Injected content via Sentry wizard below
+
+const {withSentryConfig} = require('@sentry/nextjs');
+
+module.exports = withSentryConfig(
+  module.exports,
+  {
+    // For all available options, see:
+    // https://github.com/getsentry/sentry-webpack-plugin#options
+
+    // Suppresses source map uploading logs during build
+    silent: true,
+    org: 'sentry',
```

Comment:
```suggestion
    silent: !process.env.CI,
```

Annoying during build and locally but useful on Vercel and github.